packages/pluggableWidgets/video-player-web/src/ui/VideoPlayer.css

```diff
@@ -63,3 +38,8 @@
 .hasError {
     display: flex;
 }
+
+.widget-video-player-preview-image {
+    width: 100%;
+    height: 100%;
+}
```

Comment:
Did this fix the resizing issue? Because right now the preview svgs still have hardcoded dimensions on both the svg and rect

Reply:
It works.